src/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEnumerator.WinRT.cs

```diff
@@ -1,49 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Will we ever need to bring this implementation back? That seems like an unclear answer yet, so should we entirely remove this file?

Reply:
Answered below.